<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1189</width>
    <height>785</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="SpriteEditor">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>640</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="yAxisTrans">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/yAxisTrans.png</normaloff>:/yAxisTrans.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>81</width>
      <height>71</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="xAxisTrans">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>70</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/xAxisTrans.png</normaloff>:/xAxisTrans.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>81</width>
      <height>71</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="rotateTrans">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>140</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/rotate.png</normaloff>:/rotate.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>81</width>
      <height>71</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="colorChooser">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>210</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/colorPickerIcon.png</normaloff>:/colorPickerIcon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>81</width>
      <height>71</height>
     </size>
    </property>
   </widget>
   <widget class="QGraphicsView" name="scenesView">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>0</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="newScene">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>660</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>New</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="fromBox">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>240</y>
      <width>45</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="toBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>240</y>
      <width>45</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>240</y>
      <width>68</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>From:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>240</y>
      <width>68</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>To:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="setRange">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>210</y>
      <width>119</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Range</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1189</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="frameDock">
   <property name="minimumSize">
    <size>
     <width>91</width>
     <height>50</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <widget class="QGraphicsView" name="smallView1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
